.github/workflows/build-kata-static-tarball-ppc64le.yaml

```diff
@@ -38,6 +38,9 @@ jobs:
       - name: Adjust a permission for repo
         run: |
           sudo chown -R $USER:$USER $GITHUB_WORKSPACE
+      
+      - name: Prepare the self-hosted runner
```

Comment:
I would be better to place this step after the checkout, just opening up a room to make an urgent patch against the repo. What do you think?

Reply:
> just opening up a room to make an urgent patch against the repo.

Hi @BbolroC, I don't mind moving it to after checkout but can I ask what you mean by this? I didn't get you completely

Reply:
Let's say what if you need to do something against the source code before the real steps start? (I know it is very unlikely to happen). I just wanted to inform you of this kind of situation in which I experienced before.

Reply:
Got it, makes sense. Changes applied, Thanks @BbolroC !